### ISSUE TITLE
FTL engine not installing

### ISSUE THREAD

**Issue by JJSSS** [pi-hole/pi-hole#2680]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**

`{1}`

---
**Expected behaviour:**

`{Install to work?}`

**Actual behaviour:**

Good morning all. This is my first Pi project running on a Pi zero W. I flashed my SD card and rasbian (latest version) installed and it workes fine. After rubbish the bash script and filling out all the prompts on screen the scrip fails giving me this:

  [i] Checking for existing FTL binary...
  [✗] Downloading and Installing FTL
  Error: Unable to get latest release location from GitHub
  [✗] FTL Engine not installed
[...]

I’ve read a few posts by other members and followed the advice given. One was to reboot and to enter this code into terminal:

echo 'nameserver 1.1.1.1' > /etc/resolv.conf

On doing this I get the error:

-bash: /etc/resolv.conf: Permission denied

I tried putting sudo and sudo -s infront of the command but I still get the same.
I have also tried manually editing the file but same result. 

I have also tried putting the SD card into my computer and editing the file that way but not luck. 

I’ve tried looking on the internet and other forums in how to get into this file but I’m having now luck. As others have reported the internet no longer works unless you use the IP address so I can ping 8.8.8.8 with no problem but unable to rerun the install script.

I have also tried rerunning the script with the latest updates to the pi and without. 

Any advice you can give me would be greatly appreciated but please bare in mind that I am new to coding so full commands and instructions would be of huge benefit. Thanks in advance

**Comment by dschaper** [comment]
Are you able to `ping` 8.8.8.8 and `ping` github.com? What is the current contents of your `/etc/resolv.conf` file?

**Comment by Mcat12** [comment]
Use `sudo nano /etc/resolv.conf`

**Comment by JJSSS** [comment]
So I got home and booted the pi up again and for some reason the pings worked straight off the bat. I have no idea how or why as I didn't edit the files since this morning.

When I tried this morning the 8.8.8.8 pinged but the github.com didn't. The only thing that I did was SSH into the pi to test to see if a ssh app on my phone worked.

I have also updated and upgraded my pi and all seems to be working!

Anyway thank you for the help and I shall mark this as solved or completed if I can
